scripts/post.js

```diff
@@ -656,24 +656,51 @@ function decorateCaptions() {
 
   })
 }
+
+function startsWithSearch(url, urlList){
+  var flag = false;
+  if(urlList){
+    urlList.forEach((link) => {
+      if(url.startsWith(link)){ flag = true; }
+    });
+  }
+  return flag;
```

Comment:
You don't have to loop through all, you can return directly with `find()` (or `some()`):
```suggestion
  return urlList && urlList.find((link) => url.startsWith(link));
```

Reply:
true! thanks a ton for these tips!